alicat/serial.py

```diff
@@ -285,6 +439,99 @@ def set_pressure(self, pressure, retries=2):
             self._set_control_point('pressure', retries)
         self._set_setpoint(pressure, retries)
 
+    def hold(self, retries=2):
+        """Override command to issue a valve hold.
+
+        For a single valve mass flow/pressure controller, hold the valve at the present value.
+        For a dual valve mass flow controller, hold the valve at the present value.
+        For a dual valve pressure controller, close both valves.
+        """
+        self._test_controller_open()
+        command = '{addr}$$H\r'.format(addr=self.address)
+        self._write_and_read(command, retries)
+
+    def cancel_hold(self, retries=2):
+        """Cancel valve hold."""
+
+        self._test_controller_open()
+        command = '{addr}$$C\r'.format(addr=self.address)
+        self._write_and_read(command, retries)
+
+    def read_PID(self, retries=2):
```

Comment:
You *might* get yelled at by `pycodestyle` for caps in the function name. The general rule is code style supersedes acronym capitalization, so it could push for `read_pid` instead.

Personally, I think it's fine either way.